lizard_auth_server/views_api_v2.py

```diff
@@ -179,10 +178,11 @@ def form_valid(self, form):
 
         """
         # The JWT message is validated; now check the message's contents.
-        if (('username' not in form.cleaned_data) or
-            ('password' not in form.cleaned_data)):
+        if ((('username' not in form.cleaned_data) or
```

Comment:
Nou ben je een haakjesfanaat geworden. Er staat één onnodig paar in. (Oftewel: de originele code is nog steeds de goede versie volgens mij).